src/vcpkg/platform-expression.cpp

```diff
@@ -541,6 +544,8 @@ namespace vcpkg::PlatformExpression
                         case Identifier::osx: return true_if_exists_and_equal("VCPKG_CMAKE_SYSTEM_NAME", "Darwin");
                         case Identifier::uwp:
                             return true_if_exists_and_equal("VCPKG_CMAKE_SYSTEM_NAME", "WindowsStore");
+                        case Identifier::xbox:
+                            return true_if_exists_and_equal("Z_VCPKG_IS_XBOX", "true");
```

Comment:
We do not have a specific CMAKE_SYSTEM_NAME for xbox (it's just a variant of "Windows"). As such, there's no clear way to have the expression evaluator know from the existing VCPKG port variables that it should return "1" for the new ``xbox`` variable.

My proposal here is to have my triplets add:

```
set(Z_VCPKG_IS_XBOX "true")
```

```